src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, DATE_FORMAT);
+        LocalDate toDate = LocalDate.parse(dateTo, DATE_FORMAT);
+
+        StringBuilder report = new StringBuilder();
+        report.append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo)
+                .append(System.lineSeparator());
+
+        for (String name : names) {
+            int totalSalary = 0;
+            for (String emloyeeData : data) {
+                String[] parts = emloyeeData.split(" ");
+                LocalDate date = LocalDate.parse(parts[0], DATE_FORMAT);
```

Comment:
add constants for indexes. Like DATE_INDEX, NAME_INDEX

![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/88092fe6-12f2-4e87-b36f-076a528ee710)